recipes/uutils/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "uutils-coreutils" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/uutils/coreutils/archive/{{ version }}.tar.gz
+  sha256: 1bdc2838e34b6ca8275cb8e3bac5074ca59158ea077e3195fb91ec0cc6b594f6
+  patches:
+    - 0001-FIX_BUILDDIR_IN_MAKE.patch  # [not win]
+    # Fixes long command and quotes problems
+    - 0002-SHORTEN_SH_COMMANDS.patch
+
+build:
+  number: 0
+  ignore_run_exports:
+    - libstdcxx-ng  # [linux]
+  ignore_run_exports_from:
+    - {{ compiler('cxx') }}  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - make
+    - cargo-bundle-licenses
```

```suggestion
requirements:
  build:
    - {{ compiler('cxx') }}
    - {{ compiler('rust') }}
    - make
    - cargo-bundle-licenses
  run_constrained:
    - coreutils ==999999999
    - gnu-coreutils ==999999999
```
I assume that your package clobbers files from these packages, so they should never be installed into the same environment.

---

Thank you! I didn't realize that `run_constrained`needs to be added in this `meta.yaml`.